src/object.c

```diff
@@ -424,12 +417,390 @@ int paint_path_entry(int flags, int ebx, int ecx, int edx, rct_drawpixelinfo* dp
 		{
 			if ((flags & 0xFF) <= 1)//0
 			{
-				uint8_t* ebp = esi + 0xE;
-				((rct_string_id*)esi)[0] = object_get_localised_text(&ebp, ecx, ebx, 0);
+				uint8_t* ebp = (uint8_t*)(esi + 0x1C2);
```

Comment:
I will do that. Also, watch out, because esi is something different when the flags value is different.